corehq/ex-submodules/casexml/apps/phone/data_providers/case/clean_owners.py

```diff
@@ -78,37 +71,37 @@ def get_payload(self):
             # todo: see if we can avoid wrapping - serialization depends on it heavily for now
             case_batch = filter(
                 partial(case_needs_to_sync, last_sync_log=self.restore_state.last_sync_log),
-                [CommCareCase.wrap(doc) for doc in get_docs(CommCareCase.get_db(), ids)
-                 if not is_deleted(doc)]
+                [case for case in CaseAccessors(self.restore_state.domain).get_cases(ids)
+                 if not case.is_deleted]
```

Comment:
Taking a closer look at this and the `case_needs_to_sync` function we may want to branch the logic here so that we can't take more advantage of the SQL db otherwise we're hitting the DB again for each case. Probably not in this PR but I think we could do the `modified_since_sync` check as part of the initial query.